Makefile

```diff
@@ -63,18 +65,17 @@ do-push-%:
 	docker tag $(PREFIX)/metrics-server-$*:$(GIT_COMMIT) $(PREFIX)/metrics-server-$*:$(VERSION)
 	docker push $(PREFIX)/metrics-server-$*:$(VERSION)
 
-	# push alternate tags
-ifeq ($*,amd64)
-	# TODO: Remove this and push the manifest list as soon as it's working
-	docker tag $(PREFIX)/metrics-server-$*:$(VERSION) $(PREFIX)/metrics-server:$(VERSION)
-	docker push $(PREFIX)/metrics-server:$(VERSION)
-endif
-
 # do build and then push a given official image
-sub-push-%: container-% do-push-% ;
+sub-push-%: container-% do-push-%;
+
+# push the fat manifest
+push-all:
+	docker manifest create --amend $(PREFIX)/metrics-server:$(VERSION) $(shell echo $(ALL_ARCHITECTURES) | sed -e "s~[^ ]*~$(PREFIX)/metrics-server\-&:$(VERSION)~g")
+	@for arch in $(ALL_ARCHITECTURES); do docker manifest annotate --arch $${arch} $(PREFIX)/metrics-server:$(VERSION) $(PREFIX)/metrics-server-$${arch}:${VERSION}; done
```

Comment:
I'm not sure that this will work with new image promotion process https://github.com/kubernetes-sigs/metrics-server/issues/365

In this process we first build images and push them to staging repo. "Promoting" (copying from staging repo to prod one) is done in different place. If we promoted fat manifest that you propose it will still point to staging repo instead of production.

Currently we are planning to release 0.3.7 using new process. I would wait with this PR after we figure out image promotion process.

Reply:
Hmm :thinking: We are doing so for the apiserver-network-proxy and it works well with the fat manifest (see https://console.cloud.google.com/gcr/images/k8s-artifacts-prod/EU/kas-network-proxy/proxy-agent@sha256:c419394f3fa40c32352be5a6ec5865270376d4351a3756bb1893be3f28fcba32/details?tab=info)

Reply:
@serathius and it would be nice to get fat manifest in 0.3.7 :smile: But yep, if you want to do it separately I get it :smile:

Reply:
I will ask on #wg-k8s-infra slack how it should work. It would be a great if 0.3.7 has fat manifests

Reply:
Ok, I think this approach will work. I will review it again and give lgtm

Reply:
:tada: